### PR TITLE
GN-4977: add meeting details modal to inauguration meeting creation flow

### PR DESCRIPTION
**Pull request by elpoelma**
### Overview
This PR adds a meeting details modal to the inauguration meeting creation flow to allow users to specify the location and time of the meeting.
Differences compared to common meeting creation modal:
- Administrative unit is not editable
- Modal title is slightly different

Additionally, this PR also includes a modification in how the governing body (bestuurseenheid) is selected for the inauguration meeting:
- The `binding-einde` should be undefined or gte than the current date. (is this correct?)

##### connected issues and PRs:
[GN-4977](https://binnenland.atlassian.net/browse/GN-4977)

### How to test/reproduce
- Start the app
- Visit the meeting inbox
- Test if both the common meeting and inauguration meeting creation flows works as expected

### Checks PR readiness
- [x] UI: works on smaller screen sizes
- [x] UI: feedback for any loading/error states
- [x] Check cancel/go-back flows
- [x] Check database state correct when deleting/updating (especially regarding relationships)
- [x] changelog
- [x] npm lint
- [x] no new deprecations

### CODE REVIEW

File: app/routes/inbox/meetings/new.js [hunk 1]
```diff
@@ -1,22 +1,72 @@
...
+
   beforeModel() {
     if (!this.currentSession.canWrite) {
       this.router.replaceWith('inbox.meetings');
     }
   }
 
-  model() {
+  model(params) {
+    console.log(params);
```
Comment (lagartoverde) [inline app/routes/inbox/meetings/new.js:23]: console.log

**Review by lagartoverde — APPROVED** [review]
image
Getting this error that doesn't seem to affect the functionality, otherwise works good

**Comment by elpoelma** [comment]
> image Getting this error that doesn't seem to affect the functionality, otherwise works good

Ah that could be caused by this: https://github.com/duetds/date-picker/pull/87

**Comment by elpoelma** [comment]
> image Getting this error that doesn't seem to affect the functionality, otherwise works good

https://github.com/lblod/frontend-gelinkt-notuleren/pull/705 should fix the issue

**Pull request merged**